src/status_im/subs/wallet/wallet.cljs

```diff
@@ -60,11 +59,6 @@
  :<- [:wallet/wallet-send]
  :-> :token)
 
-(rf/reg-sub
```

Comment:
you sure you should delete this?

Reply:
I checked for the usage. It's an unused subscription.
If it's needed, I will revert it 👍

Reply:
No worries if it's handled. I will revert this part. 👍